static/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -210,9 +212,19 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     const {organization} = this.props;
     const provider = this.provider;
 
-    if (configurations.length) {
-      return configurations.map(integration => {
-        return (
+    if (!configurations.length) {
+      return this.renderEmptyConfigurations();
+    }
+
+    const alertText = getAlertText(configurations);
+    return (
+      <div>
```

Comment:
nit: If the div isn't being used for formatting can we switch to `<Fragment>` to indicate that explicitly?